PxWeb/appsettings.json

```diff
@@ -65,7 +65,8 @@
     "DefaultOutputFormat": "px"
   },
   "LuceneConfiguration": {
-    "IndexDirectory": "Database"
+    "IndexDirectory": "Database",
+    "UseStandardAnalyzer": true
```

Comment:
I think default should be false

Reply:
I didn't want to change how it worked, but I cant think of a use case for the StandardAnalyzer (assuming all sites have an english version). @likp what say thy?

Reply:
Since it is a new API I think we could have changes and one of the could be the the default analyzer, so I am for setting the default value to false.

Reply:
Hmm, does it need to be configurable? It somehow feels safer, to be able to turn it off, but it also complicates things ...

Reply:
I don't think we really need the `UseStandardAnalyzer` setting :-)

Reply:
We decided to remove the option.